### DISCUSSION TITLE
After logging in, builder doesn't load.

### DISCUSSION THREAD

**Discussion by stellarfinn** [Budibase/budibase#14121] [Help]
Logged in earlier today without issue. Not anymore, the loading wheel spins in the tab and I get a black screen. On a Budibase paid hosting account. 

https://github.com/Budibase/budibase/assets/99847728/4c3a62c9-c5c9-4c92-8cfb-53d3a2b8e5af

**Comment by ConorWebb96** [comment]
Hey @stellarfinn,

Can you email into support@budibase.com and provide screenshots of your console and network. We can have a more in-depth look there.

Try clearing your browser's cookies and cache and then login and see if that fixes it.
